comments/api/v1/views.py

```diff
@@ -11,6 +15,8 @@ class CommentViewSet(GenericViewSet):
     queryset = Comment.objects.all()
 
     def get_permissions(self):
+        if self.action in ["create"]:
+            return [IsAuthenticated()]
```

Comment:
authenticate thì mình nên đặt config chung trong file `settings` -> bắt buộc toàn bộ request phải xác thực và check quyền
với action nào ko cần bước check này (vd: get list/show article, list comments) thì mình custom riêng tại method `get_permission` tại class view như này nhé, set giá trị `AllowAny`, vd:
```python
def get_permissions(self):
        if self.action in ['list', 'retrieve']:
            permission_classes = [AllowAny] # Ai cũng xem được
```